pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -541,7 +532,8 @@ func (r *hibernationReconciler) nodesReady(cd *hivev1.ClusterDeployment, remoteC
 		return false, errors.New("cannot find hibernating condition")
 	}
 	// Don't delay nodeCheckWaitTime if we just discovered SyncSets have been applied
-	if hibernatingCondition.Reason != hivev1.SyncSetsAppliedReason && time.Since(hibernatingCondition.LastProbeTime.Time) < nodeCheckWaitTime {
+	if hibernatingCondition.Reason == hivev1.SyncSetsAppliedReason &&
```

Comment:
It was correct before:
- If the hibernating condition is SyncSetsApplied, *don't* return; carry on reconciling
- Otherwise, decide whether we've delayed long enough. If so, carry on reconciling; otherwise return.